util/theta_setup.sh

```diff
@@ -1,2 +1,4 @@
 export PROJECT=ClimateEnergy_3
 export CHARGE_ACCOUNT=ClimateEnergy_3
+
+source /lus/theta-fs0/projects/ccsm/acme/tools/e3sm-unified/load_latest_cime_env.sh
```

Comment:
This also broke e3sm_integration tests on Theta: https://my.cdash.org/viewTest.php?buildid=1726087. Default python is 2.7.13, this loads python 3.7.3. It appears that some parts of CIME are not fully python3 yet (python2 is okay with that if-expression): https://github.com/E3SM-Project/E3SM/blob/master/cime/scripts/lib/CIME/XML/env_batch.py#L418
```
Submit job case.test
Traceback (most recent call last):
  File "./case.submit", line 126, in <module>
    _main_func(__doc__)
  File "./case.submit", line 123, in _main_func
    mail_user=mail_user, mail_type=mail_type, batch_args=batch_args, workflow=workflow)
  File "/gpfs/mira-home/climenrg/theta/workspace/ClimateEnergy_3/E3SM_theta_next/ACME/cime/scripts/Tools/../../scripts/lib/CIME/case/case_submit.py", line 199, in submit
    custom_success_msg_functor=verbatim_success_msg)
  File "/gpfs/mira-home/climenrg/theta/workspace/ClimateEnergy_3/E3SM_theta_next/ACME/cime/scripts/Tools/../../scripts/lib/CIME/utils.py", line 1715, in run_and_log_case_status
    rv = func()
  File "/gpfs/mira-home/climenrg/theta/workspace/ClimateEnergy_3/E3SM_theta_next/ACME/cime/scripts/Tools/../../scripts/lib/CIME/case/case_submit.py", line 197, in <lambda>
    batch_args=batch_args, workflow=workflow)
  File "/gpfs/mira-home/climenrg/theta/workspace/ClimateEnergy_3/E3SM_theta_next/ACME/cime/scripts/Tools/../../scripts/lib/CIME/case/case_submit.py", line 143, in _submit
    mail_type=mail_type, batch_args=batch_args, workflow=workflow)
  File "/gpfs/mira-home/climenrg/theta/workspace/ClimateEnergy_3/E3SM_theta_next/ACME/cime/scripts/Tools/../../scripts/lib/CIME/case/case.py", line 1348, in submit_jobs
    batch_args=batch_args, dry_run=dry_run, workflow=workflow)
  File "/gpfs/mira-home/climenrg/theta/workspace/ClimateEnergy_3/E3SM_theta_next/ACME/cime/scripts/Tools/../../scripts/lib/CIME/XML/env_batch.py", line 516, in submit_jobs
    workflow=workflow)
  File "/gpfs/mira-home/climenrg/theta/workspace/ClimateEnergy_3/E3SM_theta_next/ACME/cime/scripts/Tools/../../scripts/lib/CIME/XML/env_batch.py", line 626, in _submit_single_job
    submitargs = self.get_submit_args(case, job)
  File "/gpfs/mira-home/climenrg/theta/workspace/ClimateEnergy_3/E3SM_theta_next/ACME/cime/scripts/Tools/../../scripts/lib/CIME/XML/env_batch.py", line 418, in get_submit_args
    if flag == "-n" and rval<= 0:
TypeError: '<=' not supported between instances of 'str' and 'int'
```
@jgfouca ^

Reply:
@amametjanov , I'll push a hotfix for that.